sass/components/_post.scss

```diff
@@ -1775,12 +1810,12 @@
             top: 1px;
             display: inline-block;
             margin: 0 1px 0 5px;
-            fill: inherit;
+            fill: currentColor;
```

Comment:
nit: I prefer the use of a color variable here as well as its more consistent.

Reply:
yes, it is more descriptive, but this way you are less prone to regressions, when the color changes on the parent ... it is kinda a stylistic choice